src/main/java/mate/academy/internetshop/controller/LoginController.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.internetshop.controller;
+
+import mate.academy.internetshop.annotations.Inject;
+import mate.academy.internetshop.exceptions.AuthenticationException;
+import mate.academy.internetshop.model.User;
+import mate.academy.internetshop.service.UserService;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.*;
```

Comment:
don't use auto import *

Reply:
sry, didn't notice it transformed into *